### ISSUE TITLE
Documentation for multi-line strings

### ISSUE THREAD

**Issue by crodjer** [manastech/crystal#1110]
At http://crystal-lang.org/docs/syntax_and_semantics/literals/string.html, the following:

```crystal
"hello
      world" # same as "hello      \nworld"
```
should probably be this:

```crystal
"hello
      world" # same as "hello\n      world"
```

**Comment by bcardiff** [comment]
Thanks! It has already been fixed at https://github.com/manastech/crystal/pull/1032
The gitbook needs to be updated still to last version.

**Comment by crodjer** [comment]
Ahh, alright. I couldn't find the source code or an existing issue for
this. Figured, this is the best I could do. Maybe I should have raised it
on IRC.

On Thu, Aug 6, 2015 at 10:06 AM, Brian J. Cardiff <notifications@github.com>
wrote:

> Closed #1110 <https://github.com/manastech/crystal/issues/1110>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/manastech/crystal/issues/1110#event-374920290>.
>



-- 
Thanks
Rohan Jain

**Comment by bcardiff** [comment]
It's better to have duplicates than missed bugs. No worries :-)

Thanks again
